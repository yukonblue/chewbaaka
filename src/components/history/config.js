/**
 * history/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 07, 2020
 * Updated  : Aug 25, 2021
 */

export const config = {
  headMeta: {
    title: "History of the cheetah | Learn about cheetahs",
    description: "Learn about the history and evolution of the cheetah.",
    keywords: [
      "cheetah history",
      "cheetah evolution",
      "cheetah lineage",
      "cheetah population",
      "cheetah range",
    ]
  },
  pageProps: {
    coverImageNamePrefix: "History_Page_Cover_Image",
    title: "History",
    subtitle: "Learn about the evolution and migration of the species, its relationship with man throughout history, and its population and ranges.",
    pageMenuItems: [
      {
        title: "Evolution",
        tocImageFilename: "history_page_toc_menu_img_01-min.jpg"
      },
      {
        title: "Cheetah and Man",
        tocImageFilename: "history_page_toc_menu_img_02-min.jpg"
      },
      {
        title: "Range and Population",
        tocImageFilename: "history_page_toc_menu_img_03-min.jpg"
      }
    ],
    pageTailNavMenu: {
      prevPage: {
        label: "Home",
        href: "/"
      },
      nextPage: {
        label: "Biology",
        href: "/biology/"
      }
    }
  },
  contentPageIntro: {
    title: "History of the Cheetah",
    content: "The evolution and history of the cheetah are just as remarkable and " +
          "interesting as the species itself. Its evolution goes back to nearly " +
          "7 million years ago in time, and spanned across almost all the continents on the planet. " +
          "Its history with human interactions goes as far back as to 32000 BCE, and " +
          "the cheetah has been a symbol of significant meaning in different civilizations and " +
          "cultures across thousands of years of human history. Even up to this day, " +
          "the cheetah continues to be considered as a spiritual animal that manifests " +
          "as a symbol of a multitude of positive characteristics, such as adaptability, " +
          "persistence, and focus.",
    image: {
      filenamePrefix: "History_Page_Intro_Section_Background",
      aspectRatio: {
        width: 861,
        height: 420
      }
    }
  },
  contentPageSections: {
    section_Evolution: {
      intro: {
        title: "Evolution",
        content: "Although many would think the cheetah had originated on the continent of Africa, " +
        "the oldest fossils place  cheetahs in North America in what is now Texas, Nevada and Wyoming. " +
        "Cheetahs were common throughout Asia, Africa, Europe and North America until the end of the last " +
        "Ice Age, about 10,000 years ago, when massive climatic changes caused large numbers of mammals to disappear."
      },
      subsections: {
        subsection_EvolutionOfCats: {
          title: "Evolution of Cats"
        },
        subsection_CheetahEvolution: {
          title: "Cheetah Evolution"
        }
      },
    },
    section_CheetahAndMan: {
      intro: {
        title: "Cheetah and Man",
        content: "The history of cheetah-human relationship goes back in millennia, " +
        "and archeological artifacts uncovered throughout the world suggest that " +
        "human interactions with the species goes hand in hand with its migration and evolution. " +
        "The relationship between cheetahs and us have been both profound and complex, " +
        "as the species have been revered, utilized, displayed, and exploited throughout human history."
      },
      subsections: {
        subsection_RelationshipsWithMan: {
          title: "Relationships with Man",
          contents: {
            paragraph_Cheetah_And_Man_01:
              'Traditional African healers and witchdoctors used cheetah foot bones in spiritualistic rituals to symbolize fleet-footedness and speed. ' +
              'They used bones from a wide variety of animals and techniques varied from tribe to tribe. Objects ' +
              'represented a person, a thing or mood from the past, present or future, and are known as Divinatory sets.',
            paragraph_Cheetah_And_Man_02:
              'During the time of the Egyptian pharaohs, the cheetah was considered a goddess named ' +
              '"Mafdet". Pharaohs kept cheetahs as close companions, ' +
              "symbolic of Mafdet's protection of the royal throne.",
            paragraph_Cheetah_And_Man_03:
              'The San of Southern Africa ate cheetah meat as a symbol for speed, but it was not a main food in their diet.',
            paragraph_Cheetah_And_Man_04:
              'Kings wore cheetah skins for dignity. Trade in cheetah skin only started after European explorers began ' +
              'requesting them.',
          }
        },
        subsection_CheetahsInSports: {
          title: "Cheetahs In Sports",
          contents: {
            paragraph_The_Hunting_Leopard_01:
              'In c1700 BC, the Egyptians were the first to tame cheetahs. They admired the cheetah for its speed, ' +
              'hunting ability and beauty. They honored cheetahs as symbols of royalty and prestige.',

            paragraph_The_Hunting_Leopard_02:
              'The swiftest animal on earth became a cherished hunting companion of Pharaohs and royalty ' +
              'throughout Europe, Asia and India. Until the early 1900s, ownership of cheetahs was as important ' +
              'to these nobles as their love for gold. Cheetahs hunt by sight so they excelled in the sport ' +
              'known as "coursing". Hunts organized by royalty and nobles were for the challenge of sport, ' +
              'not for food. Hunts represented power and prestige.',

            paragraph_The_Hunting_Leopard_03:
              'By the 1500s the popularity of the cheetah as a hunting companion rivaled that of the dog. ' +
              'Cheetahs, the most easily tamed of the big cats, were caught, tamed and trained. Adults were ' +
              'used because cubs had not learned how to hunt. Tamed cheetahs formed a strong bond with their keepers.',

            paragraph_The_Hunting_Leopard_04:
              'Each cheetah rode to the hunt by horseback or on a cart. Its eyes were coved with a hood and uncovered ' +
              'when prey were sighted. The cat was released to chase down the prey then rewarded with meat fed from a ' +
              'wooden spoon.',

            paragraph_The_Hunting_Leopard_05:
              'Although cherished, pampered cheetahs were loved to near extinction and taken from the wild in great numbers. ' +
              'By the early 1900s, India and Iran were importing African cheetahs for the sport of "coursing", as their own ' +
              'wild population became too small.'
          }
        },
        subsection_CheetahsInArt: {
          title: "Cheetahs In Art",
          contents: {
            paragraph_Cheetah_In_Art_01:
              'Sumerians (Iraqi ancestors) were the first to use cheetahs in art.',

            paragraph_Cheetah_In_Art_02:
              'The cheetah and leopard have often been mistaken for each other. ' +
              'Cheetahs in early art were frequently called "panther" ' +
              'or "hunting leopard". Early artists called the true leopard "pardus" ' +
              'and described it as a cross between a cheetah (panther), and the lion (leo).',

            paragraph_Cheetah_In_Art_03:
              'Middle Age and Renaissance artists began drawing cheetahs and other ' +
              'animals more life-like. During this time of scientific exploration, ' +
              'detailed descriptions of animals provided incentive for reality in art. ' +
              'Artists included cheetahs more often in their paintings as the trade in ' +
              'live animals increased.',

            paragraph_Cheetah_In_Art_04:
              'Artists always pictured cheetahs as animals of speed and royalty.'
          }
        }
      }
    },
    section_PopulationAndRange: {
      intro: {
        title: "Range and Population",
        content: "A combination of factors such as game hunting, retaliatory killing, " +
                "habitat encroachment have caused the sharp decline of the global cheetah " +
                "population over the last century, and driven the species out of much of their " +
                "historical range. One of the last strongholds of the species today remain in " +
                "southern Africa, spanning over Namibia and Botswana, which together " +
                "holds two thirds of the species' population."
      },
      subsections: {
        subsection_Namibia: {
          title: "Namibia - Cheetah Capital of the World",
          contents: {
            Cheetah_Capital_Of_The_World: {
              title: "Namibia",
              subtitle: "Cheetah Capital of the World",
              content: {
                paragraph_Cheetah_Capital_Of_The_World_01:
                  "Namibia has the world's largest cheetah population. " +
                  "Approximately 3000 cheetahs share the land with humans, " +
                  "livestock, and wildlife.",

                paragraph_Cheetah_Capital_Of_The_World_02:
                  "Today, the status of the Namibian cheetah is stabilizing. " +
                  "During the 1980s, the population of Namibian cheetahs " +
                  "declined by half. In the 10-year period, nearly " +
                  "7,000 cheetahs were removed from the wild.",

                paragraph_Cheetah_Capital_Of_The_World_03:
                  "In most countries where cheetahs live, their numbers " +
                  "have been reduced to critical levels. The cheetah's " +
                  "survival worldwide is in human hands."
              }
            },
            Namibia_Home_For_The_Future: {
              title: "Namibia",
              subtitle: "Hope for the Future",
              content: {
                paragraph_01:
                  "The greatest hope for the cheetah's survival lies in the " +
                  'pristine countryside of Namibia. Nearly 1000 Namibian farmers ' +
                  "control the fate of the country's cheetahs and the land they live."
              }
            }
          }
        },
        subsection_RoadToExtinction: {
          title: "The Road to Extinction",
          contents: {
            The_Road_To_Extinction: {
              title: "The Road to Extinction?",
              subtitle: "1990",
              content: {
                paragraph_The_Road_To_Extinction_01:
                  "World development, industrialization, automobiles, and airplanes...",

                paragraph_The_Road_To_Extinction_02:
                  "Through the 1900s, man's inventions seemed limitless, yet there is " +
                  "nothing man-made that rivals the speed and efficiency of the cheetah.",

                paragraph_The_Road_To_Extinction_03:
                  "Scientists classified the cheetah as Acinonyx jubatus, yet nobles still " +
                  'referred to it as the "hunting leopard". By the end of the 1800s, cheetahs ' +
                  'were a rarity in Asia Minor and Arabia because of their use in the sport ' +
                  'of "coursing".',

                paragraph_The_Road_To_Extinction_04:
                  'Although it appeared that the cheetahs had a large range, their numbers ' +
                  'within that range existed in small pockets. Cheetahs, farmers and their livestock ' +
                  'all preferred open grasslands for their habitat. Increasing agricultural development ' +
                  'and new settlements played havoc with remaining cheetah populations.',
              }
            },
            Where_Did_The_Cheetah_Go: {
              title: "Where Did the Cheetah Go?",
              subtitle: "1975",
              content: {
                paragraph_Where_Did_The_Cheetah_Go_01:
                  'As human population grew, the amount of land devoted to livestock farming ' +
                  'steadily increased. Livestock filled the open land where cheetahs roamed. ' +
                  'Natural prey became scarce. Farmers kill other large predators. Although ' +
                  'game reserves protected them, cheetahs could not compete against hyenas ' +
                  'and lions. Farmlands offered cheetahs a safe haven, but they sometimes killed livestock.',

                paragraph_Where_Did_The_Cheetah_Go_02:
                  'Farmers saw cheetahs more frequently and thought their numbers had increased. ' +
                  'Cheetahs took the blame for most predator-related livestock losses. ' +
                  'Farmers killed cheetahs by the thousands as pests or to sell their skins ' +
                  'to the fur trade.',

                paragraph_Where_Did_The_Cheetah_Go_03:
                  'By 1975, researches realized that the cheetah was in trouble. CITES ' +
                  '(Convention on International Trade of Endangered Species) placed the cheetah ' +
                  'on Appendix I, making international trade in live cheetah or cheetah products illegal. ' +
                  'Local laws supported CITES in many countries where cheetahs still lived. ' +
                  'Researchers began looking for ways to encourage the growth of cheetah populations ' +
                  'through land management practices.'
              }
            }
          }
        }
      }
    }
  }
};
