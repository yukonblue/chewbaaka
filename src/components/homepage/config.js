/**
 * homepage/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 28, 2020
 * Updated  : Aug 17, 2021
 */

export const config = {
  components: {
    landingPageGateway: {
      constants: {
        landingPageGatewayTitle: "Learn about cheetahs",
        landingPageGatewaySubtitle: "Explore the Cheetah Museum at Cheetah Conservation Fund"
      },
    }
  }
}
