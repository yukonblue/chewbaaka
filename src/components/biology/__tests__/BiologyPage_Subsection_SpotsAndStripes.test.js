/**
 * BiologyPage_Subsection_SpotsAndStripes.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 19, 2020
 * Updated  : Aug 20, 2021
 */

import React from 'react'

import { render } from '@testing-library/react'

import { config } from '../config'

import renderer from 'react-test-renderer'

import BiologyPageSubsectionSpotsAndStripes from '../BiologyPage_Subsection_SpotsAndStripes'

const sectionConfig = config.contentPageSections["section_Physiology"];

test('renders BiologyPageSubsectionSpotsAndStripes component', () => {
  /**
   * Disable test because we currently use require.context
   * and it doesn't work in Jest.
   */
  // render(
  //   <BiologyPageSubsectionSpotsAndStripes
  //     sectionConfig={sectionConfig}
  //   />
  // );
});

test('BiologyPageSubsectionSpotsAndStripes component snapshot', () => {
  /**
   * Disable test because we currently use require.context
   * and it doesn't work in Jest.
   */
  // const tree = renderer
  //   .create(
  //     <BiologyPageSubsectionSpotsAndStripes
  //       sectionConfig={sectionConfig}
  //     />
  //   ).toJSON();
  // expect(tree).toMatchSnapshot();
});
