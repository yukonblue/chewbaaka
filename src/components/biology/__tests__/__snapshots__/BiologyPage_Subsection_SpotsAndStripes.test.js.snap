// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BiologyPageSubsectionSpotsAndStripes component snapshot 1`] = `
<section
  className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
  style={
    Object {
      "backgroundColor": "#98DBC6",
    }
  }
>
  <h3
    className="ContentPageSubsectionTitle"
  >
    Spots and Stripes
  </h3>
  <div>
    <div
      className="ContentPageSubsectionPartContainer"
    >
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        Adult cheetahs are easily distinguished from other cats by their coat patterns. The color and spots are a form of camouflage. This helps cheetahs hunt prey and hide from other predators.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        Distinctive black tear stripes run from the eyes to the mouth. The stripes are thought to protect the eyes from the sun’s glare. It is believed they have the same function as a rifle scope, helping cheetahs focus on their prey.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        Until three months of age, cheetah cubs have a thick-silvery grey mantle that runs down their back. The mantle helps camouflage the cubs by blending them into the shadows and grass. It also provides protection from sun and rain.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        With their mantle, cubs look like an aggressive animal called a honey badger. This may deter predators such as lions, hyenas, and eagles from attempting to kill them. This is known as “mimicry”.
      </p>
      <img
        alt="What is camouflage?"
        className="FactBannerImage FullWidth DisplayBlock VerticalCushionPadding Centered FactBannerMediumDimension"
        height={480}
        src="What_is_Camouflage_L-min.png"
        width={960}
      />
    </div>
    <div
      className="ContentPageSubsectionPartContainer"
    >
      <div
        className="Centered FullWidth"
        style={
          Object {
            "width": 360,
          }
        }
      >
        <div
          className="ImageViewOuterContainer"
          style={
            Object {
              "maxWidth": 360,
            }
          }
        >
          <div
            className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadowStyles ImageViewFamilyContainerBackgroundStyles"
          >
            <div
              className="ImageViewImgContainer"
            >
              <img
                alt="Mantle on cheetah cub's back provides a form of camouflage that's critical to its survival."
                className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                height={360}
                src="Cheetah_Mantle-min.jpg"
                width={360}
              />
            </div>
            <div
              className="ImageViewFamilyCaptionContainerStyles"
            >
              <div
                className="CaptionContainer"
                data-testid="CaptionComponentTestId"
                style={
                  Object {
                    "maxWidth": 360,
                  }
                }
              >
                <div
                  className="IconContainer"
                >
                  <i
                    aria-hidden="true"
                    className="image icon"
                    onClick={[Function]}
                  />
                </div>
                <p
                  className="ContentPageCaptionTextSize CaptionText"
                >
                  Mantle on cheetah cub's back provides a form of camouflage that's critical to its survival.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="VerticalCushionPaddingTopLarge"
      >
        <div
          className="Centered FullWidth"
          style={
            Object {
              "width": 720,
            }
          }
        >
          <div
            className="ImageViewOuterContainer"
            style={
              Object {
                "maxWidth": 720,
              }
            }
          >
            <div
              className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadowStyles ImageViewFamilyContainerBackgroundStyles"
            >
              <div
                className="ImageViewImgContainer"
              >
                <img
                  alt="The cheetah's mantle also provides a form of “mimicry” that it can use to deter predators."
                  className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                  height={360}
                  src="cheetah_and_honey_badger_L-min.jpg"
                  width={720}
                />
              </div>
              <div
                className="ImageViewFamilyCaptionContainerStyles"
              >
                <div
                  className="CaptionContainer"
                  data-testid="CaptionComponentTestId"
                  style={
                    Object {
                      "maxWidth": 720,
                    }
                  }
                >
                  <div
                    className="IconContainer"
                  >
                    <i
                      aria-hidden="true"
                      className="image icon"
                      onClick={[Function]}
                    />
                  </div>
                  <p
                    className="ContentPageCaptionTextSize CaptionText"
                  >
                    The cheetah's mantle also provides a form of “mimicry” that it can use to deter predators.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        className="BigCatSpotsIllustrationPageSubsectionWrapper"
      >
        <div
          className="ContentPageSubsectionPartContainer"
        >
          <div
            className="BigCatSpotsIllustrationOuterContainer"
          >
            <div>
              <h4
                className="IllustrationFontName IllustrationTitleFontSize"
              >
                What Cat am I?
              </h4>
              <h5
                className="IllustrationFontName IllustrationSubtitleFontSize"
              >
                Identify the cat by looking at coat patterns and colors.
              </h5>
              <div
                className="ui four column grid"
              >
                <div
                  className="row"
                >
                  <div
                    className="column"
                  >
                    <div
                      className="BigCatSpotsIllustrationItemContainer"
                    >
                      <div
                        className="BigCatSpotsIllustrationTextRevealDetailedContainer"
                      >
                        <div
                          style={
                            Object {
                              "position": "absolute",
                            }
                          }
                        >
                          <div
                            className="ui fade reveal"
                          >
                            <div
                              className="ui visible content"
                            >
                              <div
                                className="TextRevealDetailedCoverContainer"
                              >
                                <img
                                  className="ui image"
                                  height={200}
                                  src="pelt_spot_tiger-min.jpg"
                                  width={200}
                                />
                              </div>
                            </div>
                            <div
                              className="ui hidden content"
                            >
                              <div
                                className="TextRevealDetailedContentContainer"
                                style={
                                  Object {
                                    "backgroundColor": "green",
                                  }
                                }
                              >
                                <img
                                  alt="Tiger"
                                  height={160}
                                  src="cat_thumbnail_tiger-min.jpg"
                                  width={200}
                                />
                                <p
                                  className="TextRevealDetailedCaptionText"
                                >
                                  Tiger
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <p
                        className="ContentPageCaptionTextSize"
                      >
                        I live in well-watered grasslands, scrubs, woodlands, or reed beds.
                      </p>
                    </div>
                  </div>
                  <div
                    className="column"
                  >
                    <div
                      className="BigCatSpotsIllustrationItemContainer"
                    >
                      <div
                        className="BigCatSpotsIllustrationTextRevealDetailedContainer"
                      >
                        <div
                          style={
                            Object {
                              "position": "absolute",
                            }
                          }
                        >
                          <div
                            className="ui fade reveal"
                          >
                            <div
                              className="ui visible content"
                            >
                              <div
                                className="TextRevealDetailedCoverContainer"
                              >
                                <img
                                  className="ui image"
                                  height={200}
                                  src="pelt_spot_jaguar-min.jpg"
                                  width={200}
                                />
                              </div>
                            </div>
                            <div
                              className="ui hidden content"
                            >
                              <div
                                className="TextRevealDetailedContentContainer"
                                style={
                                  Object {
                                    "backgroundColor": "green",
                                  }
                                }
                              >
                                <img
                                  alt="Jaguar"
                                  height={160}
                                  src="cat_thumbnail_jaguar-min.jpg"
                                  width={200}
                                />
                                <p
                                  className="TextRevealDetailedCaptionText"
                                >
                                  Jaguar
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <p
                        className="ContentPageCaptionTextSize"
                      >
                        I'm from South America and live in the jungle.
                      </p>
                    </div>
                  </div>
                  <div
                    className="column"
                  >
                    <div
                      className="BigCatSpotsIllustrationItemContainer"
                    >
                      <div
                        className="BigCatSpotsIllustrationTextRevealDetailedContainer"
                      >
                        <div
                          style={
                            Object {
                              "position": "absolute",
                            }
                          }
                        >
                          <div
                            className="ui fade reveal"
                          >
                            <div
                              className="ui visible content"
                            >
                              <div
                                className="TextRevealDetailedCoverContainer"
                              >
                                <img
                                  className="ui image"
                                  height={200}
                                  src="pelt_spot_cheetah-min.jpg"
                                  width={200}
                                />
                              </div>
                            </div>
                            <div
                              className="ui hidden content"
                            >
                              <div
                                className="TextRevealDetailedContentContainer"
                                style={
                                  Object {
                                    "backgroundColor": "green",
                                  }
                                }
                              >
                                <img
                                  alt="Cheetah"
                                  height={160}
                                  src="cat_thumbnail_cheetah-min.jpg"
                                  width={200}
                                />
                                <p
                                  className="TextRevealDetailedCaptionText"
                                >
                                  Cheetah
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <p
                        className="ContentPageCaptionTextSize"
                      >
                        I can live anywhere but prefer savannas.
                      </p>
                    </div>
                  </div>
                  <div
                    className="column"
                  >
                    <div
                      className="BigCatSpotsIllustrationItemContainer"
                    >
                      <div
                        className="BigCatSpotsIllustrationTextRevealDetailedContainer"
                      >
                        <div
                          style={
                            Object {
                              "position": "absolute",
                            }
                          }
                        >
                          <div
                            className="ui fade reveal"
                          >
                            <div
                              className="ui visible content"
                            >
                              <div
                                className="TextRevealDetailedCoverContainer"
                              >
                                <img
                                  className="ui image"
                                  height={200}
                                  src="pelt_spot_leopard-min.jpg"
                                  width={200}
                                />
                              </div>
                            </div>
                            <div
                              className="ui hidden content"
                            >
                              <div
                                className="TextRevealDetailedContentContainer"
                                style={
                                  Object {
                                    "backgroundColor": "green",
                                  }
                                }
                              >
                                <img
                                  alt="Leopard"
                                  height={160}
                                  src="cat_thumbnail_leopard-min.jpg"
                                  width={200}
                                />
                                <p
                                  className="TextRevealDetailedCaptionText"
                                >
                                  Leopard
                                </p>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                      <p
                        className="ContentPageCaptionTextSize"
                      >
                        I can live anywhere that is rich in prey and has thick bush for cover. I'm often seen resting on trees.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
